setup.py

```diff
@@ -43,10 +43,7 @@
     ],
     extras_require={
         "lint": [
-            "black>=19.10b0,<=20.8",
-            # click>=8.1.0 is incompatible with black.
-            # https://github.com/psf/black/issues/2964
```

Comment:
Black >= 22.3 seems to have resolved the compatibility issue with click: https://github.com/psf/black/issues/2964#issue-1183788875